pkg/manager/cri.go

```diff
@@ -33,6 +33,7 @@ func podSandboxMetadata(in *types.PodSandboxInfo) *kubeapi.PodSandboxMetadata {
 		Uid:       in.PodID,
 		Namespace: in.Config.Namespace,
 		Attempt:   in.Config.Attempt,
+		Tenant:    in.Config.Tenant,
```

Comment:
please add a comment here. for now, this info is passed down to the runtime service ONLY for CNI to support multi-tenant networking. this will clarify the purpose -- runtime logic probably will not consume this filed even for a long till further design changes.

Reply:
will add a comment as suggested.